package.json

```
@@ -23,10 +23,13 @@
   "scripts": {
     "prepare": "husky",
     "build": "npx hardhat compile",
+    "upgrade": "hardhat run ./scripts/upgrade.ts",
```

There are 2 upgrade types: VoucherHub upgrade and vouchers upgrade, so we need to make the difference.
```suggestion
    "upgrade": "hardhat run ./scripts/upgrade-hub.ts",
```

---

yeah it's true. Nevertheless I will use camelCase 
Fix Commit: 518c157f856242f733976373307a7af503a23e39